src/services/dataRenderService.ts

```diff
@@ -332,48 +332,39 @@ export class DataRenderService {
 
             this.applyOnObjectStylesToCircleText(text);
 
-        } else {            
+        } else {
             element.selectAll(DataRenderService.CircleText.selectorName).remove();
         }
     }
 
     private applyOnObjectStylesToCircleText(text: d3Selection<SVGTextElement, number, SVGGElement, number>): void{
         text
-        .attr(SubSelectableObjectNameAttribute, AsterPlotObjectNames.Ticks.name)
-        .attr(SubSelectableDisplayNameAttribute, this.localizationManager.getDisplayName(AsterPlotObjectNames.Ticks.displayNameKey))
-        .attr(SubSelectableTypeAttribute, SubSelectionStylesType.Text)
-        .classed(HtmlSubSelectableClass, this.formatMode && this.settings.outerLine.showGridTicksValues.value);
+            .attr(SubSelectableObjectNameAttribute, AsterPlotObjectNames.Ticks.name)
+            .attr(SubSelectableDisplayNameAttribute, this.localizationManager.getDisplayName(AsterPlotObjectNames.Ticks.displayNameKey))
+            .attr(SubSelectableTypeAttribute, SubSelectionStylesType.Text)
+            .classed(HtmlSubSelectableClass, this.formatMode && this.settings.outerLine.showGridTicksValues.value);
     }
 
-    private drawArcCircles(
+    private drawInnerAndOuterCircles(
```

Comment:
Non-optimal name. `drawInnerAndOuterCircles` means that it will draw 2 circles. 
But this function logic draws one circle to determine a border - outer or inner. 
I advise the following name `drawBorderCircle`